src/core_modules/capture-core/rules/converters/inputConverter.js

```diff
@@ -6,117 +6,53 @@ import type { IConvertInputRulesValue } from 'capture-core-utils/rulesEngine/rul
 
 const dateMomentFormat = 'YYYY-MM-DD';
 
-class RulesValueConverter implements IConvertInputRulesValue {
-    convertText(value: any): string {
-        return value || '';
-    }
-
-    convertLongText(value: any): string {
-        return value || '';
-    }
-
-    convertLetter(value: any): string {
-        return value || '';
-    }
-
-    convertPhoneNumber(value: any): string {
-        return value || '';
-    }
-
-    convertEmail(value: any): string {
-        return value || '';
-    }
-
-    convertBoolean(value: any): boolean | string {
-        return (value || value === false) ? value : '';
-    }
-
-    convertTrueOnly(value: any): boolean | string {
-        return (value || value === false) ? value : '';
-    }
-
-    convertDate(value: any): string {
+export const inputConverter: IConvertInputRulesValue = {
+    convertText: (value: ?string): ?string => (value || null),
+    convertLongText: (value: ?string): ?string => (value || null),
+    convertLetter: (value: ?string): ?string => (value || null),
+    convertPhoneNumber: (value: ?string): ?string => (value || null),
+    convertEmail: (value: ?string): ?string => (value || null),
+    convertBoolean: (value: ?boolean): ?boolean => ((value || value === false) ? value : null),
+    convertTrueOnly: (value: ?boolean): ?boolean => ((value || value === false) ? value : null),
```

Comment:
Hey,
Is the comparison with `false` necessary for the `TrueOnly` converter? Maybe the logic can be simplified into this: 

`convertTrueOnly: (value: ?boolean): ?boolean => (value || null)`

WDYT?

Reply:
I think you are right :)